apm-agent-plugins/apm-profiling-plugin/src/main/java/co/elastic/apm/agent/profiler/ProfilingConfiguration.java

```diff
@@ -60,6 +60,13 @@
         .tags("added[1.15.0]", "experimental")
         .buildWithDefault(false);
 
+    private final ConfigurationOption<Integer> recoverMode = ConfigurationOption.<Integer>integerOption()
+        .key("recover_mode")
+        .configurationCategory(PROFILING_CATEGORY)
+        .dynamic(false)
+        .tags("internal")
+        .buildWithDefault(0);
```

Comment:
This disables all stack trace recoveries by default. Is that intentional?

Reply:
This is not what I plan to push to master, this is only an analysis agent, where we want to start by testing that the recovery code is not the cause. Only if required, we will ask the user to set this to other values.
I did it as a PR just so I have the artefacts publicly available.